pkgs/by-name/lo/loadwatch/package.nix

```diff
@@ -2,27 +2,40 @@
   lib,
   stdenv,
   fetchgit,
+  autoreconfHook,
 }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation (finalAttrs: {
   pname = "loadwatch";
   version = "1.1-1-g6d2544c";
 
   src = fetchgit {
-    url = "git://woffs.de/git/fd/loadwatch.git";
-    sha256 = "1bhw5ywvhyb6snidsnllfpdi1migy73wg2gchhsfbcpm8aaz9c9b";
+    url = "https://git.sr.ht/~woffs/loadwatch";
+    hash = "sha256-K7H0lUL1suU0hOyJx8fxL9YQ23WUWt2i1WZ5uLkvHK4=";
     rev = "6d2544c0caaa8a64bbafc3f851e06b8056c30e6e";
   };
 
+  patches = [
+    # autoheader fails when template is not defined
+    ./missing-kstat-template-autoheader.patch
+    # implicit <string.h> not allowed in c99 mode
+    ./fix-c99-failure.patch
```

Comment:
Is those patches submitted upstream ?

Reply:
I don't know how to do this. The project git repo doesn't seem to explain. But AFAIU the maintainer of the git fork of the tool is the nix maintainer and is in CC here; @woffs. Perhaps he could pick these patches up, or clarify where to send them to.)